.github/workflows/build.yml

```diff
@@ -8,8 +8,19 @@ jobs:
     strategy:
       matrix:
         os: [macos-10.15, ubuntu-20.04]
+        python-version: [3.6, 3.7]
     steps:
       - name: Checkout Repository
         uses: actions/checkout@v2
-      - name: echo-test
-        run: echo 123
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install flake8 multiqc
+          python setup.py install
+      - name: Lint with flake8
+        run: |
+          flake8
```

Comment:
Please remove this from here, for the linting we will use a separate Action, take a look at one of my toy repos:  
https://github.com/AngryMaciek/ToySnake/blob/master/.github/workflows/lint.yml
Therefore we will have separate shields for `build` and `flake8` 😉

Reply:
Miniconda here is not neccesary, ofc. My point is just to have a separate action with `black` and `flake8` checks.